context/app/static/js/components/Detail/DetailLayout/utils.js

```diff
@@ -0,0 +1,12 @@
+import { capitalizeString } from 'js/helpers/functions';
+
+function getSectionFromString(s) {
+  return [s, { text: capitalizeString(s), hash: s }];
```

Comment:
I don't think this is a change in this PR, but if it's not complicated, could the lists we give to `getSectionOrder` have the intended capitalization? In the past I've been boxed in when I started like this, and then there was some name that needed non-standard capitalization.

If it's an easy change in this PR, great, or file an issue if you think it's worthwhile.